Resources/Prototypes/SS220/Entities/Objects/Consumable/Smokeables/Cigarettes/packs.yml

```
@@ -0,0 +1,180 @@
+# © SS220, An EULA/CLA with a hosting restriction, full text: https://raw.githubusercontent.com/SerbiaStrong-220/space-station-14/master/CLA.txt
+
+- type: entity
+  id: CigPackBaseSS220
+  parent: [ BaseStorageItem, BaseBagOpenClose ]
+  name: cigarette pack
+  abstract: true
+  components:
+  - type: Sprite
+    layers:
+    - state: closed
+      map: ["closeLayer"]
+    - state: open
+      map: ["openLayer"]
+      visible: false
+    - state: 1
+      map: ["1"]
+      sprite: SS220/Objects/Consumable/Smokeables/Cigaretts/Packs/visualizer.rsi
+      visible: false
+    - state: 2
+      map: ["2"]
+      sprite: SS220/Objects/Consumable/Smokeables/Cigaretts/Packs/visualizer.rsi
+      visible: false
+    - state: 3
+      map: ["3"]
+      sprite: SS220/Objects/Consumable/Smokeables/Cigaretts/Packs/visualizer.rsi
+      visible: false
+    - state: 4
+      map: ["4"]
+      sprite: SS220/Objects/Consumable/Smokeables/Cigaretts/Packs/visualizer.rsi
+      visible: false
+    - state: 5
+      map: ["5"]
+      sprite: SS220/Objects/Consumable/Smokeables/Cigaretts/Packs/visualizer.rsi
+      visible: false
+    - state: 6
+      map: ["6"]
+      sprite: SS220/Objects/Consumable/Smokeables/Cigaretts/Packs/visualizer.rsi
+      visible: false
+    - state: 7
+      map: ["7"]
+      sprite: SS220/Objects/Consumable/Smokeables/Cigaretts/Packs/visualizer.rsi
+      visible: false
+    - state: 8
+      map: ["8"]
+      sprite: SS220/Objects/Consumable/Smokeables/Cigaretts/Packs/visualizer.rsi
+      visible: false
+    - state: 9
+      map: ["9"]
+      sprite: SS220/Objects/Consumable/Smokeables/Cigaretts/Packs/visualizer.rsi
+      visible: false
+    - state: 10
+      map: ["10"]
+      sprite: SS220/Objects/Consumable/Smokeables/Cigaretts/Packs/visualizer.rsi
+      visible: false
+  - type: Tag
+    tags:
+    - CigPack
+    - Trash
+  - type: PhysicalComposition
+    materialComposition:
+      Steel: 50
+  - type: SpaceGarbage
+  - type: Item
+    size: Tiny
+    shape:
+    - 0,0,0,1
+  - type: Storage
+    grid:
+    - 0,0,4,1
+  - type: StorageFill
+    contents:
+    - id: Cigarette
+      amount: 10
+  - type: ItemCounter
+    count:
+      tags: [Cigarette]
+    composite: true
+    layerStates:
+    - 1
+    - 2
+    - 3
+    - 4
+    - 5
+    - 6
+    - 7
+    - 8
+    - 9
+    - 10
+  - type: Appearance
+
+- type: entity
+  id: CigPackENG
+  parent: CigPackBaseSS220
+  name: Friendly singularity's packet
+  description: A label on the packaging reads, Face the singularity with no fear.
+  components:
+  - type: Sprite
+    sprite: SS220/Objects/Consumable/Smokeables/Cigarettes/Packs/ENG.rsi
```

```suggestion
    sprite: SS220/Objects/Consumable/Smokeables/Cigaretts/Packs/ENG.rsi
```
Везде где есть подобный путь изменить на указанный. Сама по себе папка называется Cigaretts